index.html

```diff
@@ -685,6 +719,26 @@ <h3>Converting between <code>tiltX</code> / <code>tiltY</code> and <code>altitud
 </pre>
             </section>
         </section>
+        <section>
+            <h2><code>DeviceProperties</code> interface</h2>
+            <pre class="idl">
+                dictionary DevicePropertiesInit {
+                    long uniqueId = -1;
+                };
+
+                [Exposed=Window]
+                interface DeviceProperties {
+                    constructor(optional DevicePropertiesInit devicePropertiesInitDict = {});
+                    readonly attribute long uniqueId;
+                };
+            </pre>
+            <dl data-dfn-for="DeviceProperties" data-link-for="DeviceProperties">
+                <dt><dfn>uniqueId</dfn></dt>
+                    <dd>
+                        <p>A unique identifier for the pointing device. If the hardware supports multiple pointers, pointer events generated from pointing devices must only get a <code>uniqueId</code> if those pointers are uniquely identifiable over the session. If the pointer is uniquely identifiable, the assigned <code>uniqueId</code> to that pointing device will remain constant for the remainder of the session. <code>uniqueId</code> value of <code>1</code> MUST be reserved for events generated by the primary mouse device. The <code>uniqueId</code> value of <code>-1</code> MUST be reserved and used to indicate events whose generating device could not be identified. Like <a href="#dom-pointerevent-pointerid">pointerId</a>, to minimize the chance of fingerprinting and tracking across different pages or domains, the <code>uniqueId</code> MUST only be associated explicitly with that particular pointing device for the lifetime of the page / session, and a new randomized <code>uniqueId</code> MUST be chosen the next time that particular pointing device is used again in a new session.</p>
```

Comment:
I don't think mouse should have any special handling here.

Reply:
If we're not following pointerId on the mouse id, would it make sense to also reserve 0 for no id so people checking the truthiness will get the expected outcome, e.g. `if (uid = event?.deviceProperties.uniqueId) { /* do something */ }`

Reply:
I think it makes sense in this case to replace -1 with 0 for invalid/no id. We don't need both values right?

Reply:
I agree we don't need both.  A `0` makes more sense to me than a `-1`.

Reply:
Done

Reply:
Closing this thread; seems like there are no more comments on this topic.